--- antigen.zsh	2018-03-16 08:41:45.490803690 +0700
+++ .antigen.zsh	2018-03-16 08:39:26.227456484 +0700
@@ -46,6 +46,7 @@
     return 0
   fi
 fi
+[[ -z "$_ANTIGEN_INSTALL_FILENAME" ]] && _ANTIGEN_INSTALL_FILENAME=$(basename $0)
 [[ -z "$_ANTIGEN_INSTALL_DIR" ]] && _ANTIGEN_INSTALL_DIR=${0:A:h}
 
 # Each line in this string has the following entries separated by a space
@@ -1812,7 +1813,7 @@
 $(functions -- _antigen)
 antigen () {
   local MATCH MBEGIN MEND
-  [[ "\$ZSH_EVAL_CONTEXT" =~ "toplevel:*" || "\$ZSH_EVAL_CONTEXT" =~ "cmdarg:*" ]] && source "$_ANTIGEN_INSTALL_DIR/antigen.zsh" && eval antigen \$@;
+  [[ "\$ZSH_EVAL_CONTEXT" =~ "toplevel:*" || "\$ZSH_EVAL_CONTEXT" =~ "cmdarg:*" ]] && source "$_ANTIGEN_INSTALL_DIR/$_ANTIGEN_INSTALL_FILENAME" && eval antigen \$@;
   return 0;
 }
 typeset -gaU fpath path
