--- /opt/google/chrome-remote-desktop/chrome-remote-desktop	2020-04-04 23:35:33.608393740 +0700
+++ /opt/google/chrome-remote-desktop/chrome-remote-desktop.bak	2020-04-04 23:35:15.964823812 +0700
@@ -105,7 +105,8 @@
 SYSTEM_SESSION_FILE_PATH = "/etc/chrome-remote-desktop-session"
 
 X_LOCK_FILE_TEMPLATE = "/tmp/.X%d-lock"
-FIRST_X_DISPLAY_NUMBER = 20
+#FIRST_X_DISPLAY_NUMBER = 20
+FIRST_X_DISPLAY_NUMBER = 1
 
 # Amount of time to wait between relaunching processes.
 SHORT_BACKOFF_TIME = 5
@@ -418,8 +419,8 @@
     """Return a candidate display number for which there is currently no
     X Server lock file"""
     display = FIRST_X_DISPLAY_NUMBER
-    while os.path.exists(X_LOCK_FILE_TEMPLATE % display):
-      display += 1
+    #while os.path.exists(X_LOCK_FILE_TEMPLATE % display):
+    #  display += 1
     return display
 
   def _init_child_env(self):
@@ -701,8 +702,10 @@
     self._init_child_env()
     self._setup_pulseaudio()
     self._setup_gnubby()
-    self._launch_x_server(x_args)
-    self._launch_x_session()
+    #self._launch_x_server(x_args)
+    #self._launch_x_session()
+    display = self.get_unused_display_number()
+    self.child_env["DISPLAY"] = ":%d" % display #Pls change the quotes to plain quotes
 
   def launch_host(self, host_config, extra_start_host_args):
     # Start remoting host
